oaknational/Oak-Web-Application
# Fix/env

## Description

- creates `.env.test` with fake env vars (but needed for tests to pass)
- delete `.env.example` since now we;re already checking in `.env.test` we can use that as the template
- updates readme
- removes setting env in `jests.setup.js`

## Issue(s)

Fixes #67 

## How to test

1. Locally, rename `.env` to `.env.development.local`
2. Run tests; should pass
3. Run app; should work
